src/acb.h

```diff
@@ -509,6 +509,28 @@ acb_div_onei(acb_t z, const acb_t x)
     }
 }
 
+ACB_INLINE void
+acb_mul_powi(acb_t z, const acb_t x, slong k)
```

Comment:
Maybe use the naming convention of ``fmpzi_mul_i_pow_si``? Also, see that function for a simpler way to mod by 4.

Reply:
This is fixed in https://github.com/flintlib/flint/pull/1524/commits/e4cafd2fa5f95468b60811afc5d8050b57b21ef2